.mdlrc

```diff
@@ -1,2 +1,2 @@
-rules "~MD013", "~MD026", "~MD029", "~MD032", "~MD036", "~MD002", "~MD034", "~MD031", "~MD028"
+rules "~MD013", "~MD026", "~MD029", "~MD032", "~MD036", "~MD002", "~MD034", "~MD031", "~MD028", "~MD033", "~MD027"
```

Comment:
I disagree with `MD033`. I want to use HTML. in this case `<br>` tags. MD027 was instead giving a false positive.

Reply:
There are two other options:
* Using a backslash `\` at the end of the line 
* Using two spaces `  ` at the end of the line

See 
![CleanShot 2024-09-04 at 13 48 31@2x](https://github.com/user-attachments/assets/f75ae95b-2fc3-40d5-8b01-40f0e58cc560)
https://daringfireball.net/projects/markdown/syntax#block